robocop/run.py

```diff
@@ -248,22 +248,46 @@ def any_rule_enabled(self, checker) -> bool:
 
     def configure_checkers_or_reports(self):
         for config in self.config.configure:
-            if config.count(":") < 2:
+            if config.count(":") < 1:
                 raise robocop.exceptions.ConfigGeneralError(
                     f"Provided invalid config: '{config}' (general pattern: <rule>:<param>:<value>)"
                 )
-            rule_or_report, param, value = config.split(":", maxsplit=2)
-            if rule_or_report in self.rules:
-                rule = self.rules[rule_or_report]
-                rule.configure(param, value)
-            elif rule_or_report in self.reports:
-                self.reports[rule_or_report].configure(param, value)
+            rule_or_report = config.split(":", maxsplit=1)
+            if rule_or_report[0] in self.rules:
+                self._configure_checkers(config)
+            elif rule_or_report[0] in self.reports:
+                self._configure_reports(config)
             else:
-                similar = RecommendationFinder().find_similar(rule_or_report, self.rules)
+                similar = RecommendationFinder().find_similar(rule_or_report[0], self.rules)
                 raise robocop.exceptions.ConfigGeneralError(
-                    f"Provided rule or report '{rule_or_report}' does not exist. {similar}"
+                    f"Provided rule or report '{rule_or_report[0]}' does not exist. {similar}"
                 )
 
+    def _configure_checkers(self, config):
+        if config.count(":") < 2:
+            raise robocop.exceptions.ConfigGeneralError(
+                f"Provided invalid config: '{config}' (general pattern: <rule>:<param>:<value>)"
+            )
+        rule_name, param, value = config.split(":", maxsplit=2)
+        if rule_name in self.rules:
+            self.rules[rule_name].configure(param, value)
+
+    def _configure_reports(self, config):
+        if config.count(":") < 1:
+            raise robocop.exceptions.ConfigGeneralError(
+                f"Provided invalid config: '{config}' (general pattern: <report>:<param>:<value>)"
+            )
```

Comment:
We could define specific exception to avoid duplicating lines. ie:

```
class InvalidConfigPatternError(ConfigGeneralError):
    def __init__(self, config):
        msg = f"Provided invalid config: '{config}' (general pattern: <report>:<param>:<value>)"
        super().__init__(msg)
        
raise InvalidConfigPatternError(config)
```

Reply:
I think, simplifying the implementation solved also the issue topic...